packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts

```diff
@@ -45,6 +58,25 @@ export async function __refreshCommunityMemberships(
     },
   });
 
+  console.log(
+    `Checking ${addresses.length} addresses in ${groupsToUpdate.length} groups in ${community.id}...`,
+  );
+  const getBalancesOptions = await makeGetBalancesOptions(
+    groupsToUpdate,
+    addresses,
+  );
+  const allBalances: OptionsWithBalances[] = await Bluebird.reduce(
```

Comment:
What is the purpose of using Bluebird.reduce here?

Reply:
Originally was used to build a map object, but changed to an array.

Will change to `Promise.all`